plugins/parse_map_mixin.rb

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
https://github.com/AnchorFree/php-logger/pull/8/files#diff-3254677a7917c6c01f55212f86c57fbfR15
is enough